Admin/AbstractAdmin.php

```diff
@@ -2843,6 +2843,21 @@ public function getDashboardActions()
     }
 
     /**
+     * Setting to true will enable mosaic button for the admin screen.
+     * Setting to false will hide mosaic button for the admin screen.
+     *
+     * @param bool $isShown
+     */
+    final public function setShowMosaicButton($isShown)
+    {
+        if ($isShown) {
+            $this->listModes['mosaic'] = array('class' => 'fa fa-th-large fa-fw');
```

Comment:
The string is a duplicate of https://github.com/tim96/SonataAdminBundle/blob/db5ebaa41fb849082969729245da38cf7924d9e2/Admin/AbstractAdmin.php#L423

If one day we change the string, this might be erroneous.

Maybe use a constant?